packages/round-manager/src/features/round/ViewRoundPage.tsx

```diff
@@ -243,3 +268,43 @@ export default function ViewRoundPage() {
     </>
   );
 }
+
+type ViewGrantsExplorerButtonType = {
+  styles?: string;
+  iconStyle?: string;
+  chainId: string;
+  roundId: string | undefined;
+};
+
+function RoundName(props: { round?: Round }) {
+  return (
+    <h1 className="text-3xl sm:text-[32px] my-2">
+      {props.round?.roundMetadata?.name || "Round Details"}
+    </h1>
+  );
+}
+
+export function ViewGrantsExplorerButton(props: ViewGrantsExplorerButtonType) {
+  const { chainId, roundId } = props;
+
+  return (
+    <Button
+      type="button"
+      className={`inline-flex items-center bg-white text-xs border border-grey-100 text-grey-500 py-1.5 px-2.5 w-48 h-7 drop-shadow-sm justify-center ${props.styles}`}
+      onClick={() => {
+        redirectToGrantExplorer(chainId, roundId);
+      }}
+      data-testid="round-explorer"
+    >
+      <GrantExplorerLogo className={props.iconStyle} aria-hidden="true" />
+      View on Grants Explorer
+    </Button>
+  );
+}
+
+function redirectToGrantExplorer(chainId: string, roundId: string | undefined) {
+  const url = `${process.env.REACT_APP_GRANT_EXPLORER}/round/${chainId}/${roundId}`;
```

Comment:
add a hash after base url since the url needs to have hash routing since fleek is using ipfs to host files
example: `https://gegitcoin.on.fleek.co/#/round/5/0xa89c6f804579864e29fdbb9653f62d761d1a42fe`